From 22eb81c247b01f3712286696a9eed9f3892eeb53 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:23:16 -0400
Subject: [redhat] kabi: Adding symbol find_first_zero_bit (lib/find_next_bit.c)

Message-id: <48b5ac58df04b60695a658b0324ead36b0d9ae4f.1410510773.git.poros@redhat.com>
Patchwork-id: 95928
O-Subject: [PATCH RHEL7.1 076/233] kabi: Adding symbol find_first_zero_bit (lib/find_next_bit.c)
Bugzilla: 1062361 1071098
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

Bugzilla: 1062361,1071098
    https://bugzilla.redhat.com/show_bug.cgi?id=1062361
    https://bugzilla.redhat.com/show_bug.cgi?id=1071098

    Description:
    ============
    Bitmap helper function for finding unset bit.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/find_first_zero_bit

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_x86_64/find_first_zero_bit b/redhat/kabi/kabi-module/kabi_x86_64/find_first_zero_bit
new file mode 100644
index 0000000..e7afc57
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/find_first_zero_bit
@@ -0,0 +1,2 @@
+#1-
+0xf11543ff find_first_zero_bit vmlinux EXPORT_SYMBOL
-- 
1.7.1